### PR TITLE
feat-cart page added

### PR DESCRIPTION
**Pull request by ImrozNezami**
Cart Page
Screenshot (252)

### CODE REVIEW

File: pages/cart-page.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            <button class="remove-cart colorWhite">Remove From Cart</button>
+            <button class="add-wishlist colorDarkgray">
+              Move to Whishlist
+            </button>
+          </div>
+        </div>
+        <div class="cart-card mar-2">
+          <div class="cart-img-container">
+            <img
+              src="https://cdn.pixabay.com/photo/2016/11/19/18/06/feet-1840619__340.jpg"
```
Comment (vaibhav-mahalle) [inline pages/cart-page.html:82]: you should add the alt tag for better accessibility.

File: pages/cart-page.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+            </div>
+            <button class="remove-cart colorWhite">Remove From Cart</button>
+            <button class="add-wishlist colorDarkgray">Move to Wishlist</button>
+          </div>
+        </div>
+        <div class="cart-card mar-2">
+          <div class="cart-img-container">
+            <img
+              src="https://cdn.pixabay.com/photo/2016/11/19/18/06/feet-1840619__340.jpg"
+              class="cart-img"
```
Comment (vaibhav-mahalle) [inline pages/cart-page.html:57]: add the alt tag for better accessibility.

File: stylesheet/cart-page.css [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  font-size: 1.3rem;
+}
+.cart-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  padding-bottom: 13px;
+}
+.lg-primary {
+  width: 95%;
```
Comment (vaibhav-mahalle) [inline stylesheet/cart-page.css:121]: well structured code. keep up the good practices.

**Review by vaibhav-mahalle — APPROVED** [review]
suggested some minor changes. please look into it.

File: pages/cart-page.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+        </button>
+      </div>
+    </nav>
+    <h2 class="center-text">MY CART (2)</h2>
+    <div class="cart-container">
+      <div class="cart-cart-container">
+        <div class="cart-card mar-2">
+          <div class="cart-img-container">
+            <img
+              src="https://cdn.pixabay.com/photo/2016/11/19/18/06/feet-1840619__340.jpg"
```
Comment (nishasen) [inline pages/cart-page.html:32]: add the alt attribute

File: stylesheet/cart-page.css [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+  gap: 1.3rem;
+  background-color: var(--white);
+}
+.cart-img-container {
+  max-width: 15rem;
+}
+.cart-img {
+  width: 100%;
+  height: 100%;
+}
```
Comment (nishasen) [inline stylesheet/cart-page.css:20]: You can use your image responsive class here, created in CL

File: stylesheet/cart-page.css [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+  outline: 2px solid transparent;
+  outline-offset: 2px;
+  border: 1px solid gray;
+  border-radius: 50%;
+  font-weight: bold;
+  font-size: 1.2rem;
+  cursor: pointer;
+}
+.product-quantity {
+  border: 1px solid black;
```
Comment (nishasen) [inline stylesheet/cart-page.css:63]: use variables here

**Review by nishasen — APPROVED** [review]
Good clean code, suggested few changes, please look into it.

**Pull request merged**